docs/sphinx/quickstart.rst

```diff
@@ -28,12 +28,13 @@ unpack it or clone the repository directly using ``git``:
 .. code-block:: sh
 
     git clone https://github.com/STEllAR-GROUP/hpx.git
+    cd hpx
 
 It is also recommended that you check out the latest stable tag:
 
 .. code-block:: sh
 
-    git checkout 1.7.1
+    git checkout branch_name  # 1.7.0, 1.7.1, etc.
```

Comment:
My preference would be to keep `1.7.1` explicitly in the command. I've seen people literally copy-paste commands and wonder why `git checkout branch_name` doesn't work. It's an extreme case, but I think there's no need to make things more difficult if we can actually have a command that one can copy-paste. What do you think?

Reply:
Also, should the `cd hpx` command move into this code block instead?

Reply:
Sure, we can do it this way!